README.md

```
@@ -0,0 +1,154 @@
+<p align="center">
+    <picture>
+        <!-- Image for dark mode -->
+        <source media="(prefers-color-scheme: dark)" srcset="https://github.com/Airlock-Auth/airlock-auth/assets/47309835/f48ea20c-242d-4d79-b852-833220ea51c6">
+        <!-- Image for light mode (default image) -->
+        <img src="https://github.com/Airlock-Auth/airlock-auth/assets/47309835/aa360624-19b2-427a-839f-3e3e004181d1" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)